### PR TITLE
Fe forum page #32

### PR DESCRIPTION
**Pull request by jonquag**
Here is an updated look at the forum page. 

Having a few styling issues but I think that may sort itself out once we get some merges completed. Looking for some advice on the styling for the inline part at the bottom of the post content.

### CODE REVIEW

**Comment by jonquag** [comment]
Forum page

**Comment by jonquag** [comment]
Forum page update 2

**Comment by jonquag** [comment]
I have updated the button theme throughout the file.
to apply the styling to any buttons made add these attributes to your button element:

variant="text"
color="primary"

you may also still add styling to this button further (or to overwrite) for width and necessary padding through your own:

className={classes.button}

**Comment by jessdesr** [comment]
Do you know why so many files were modified in this PR?

**Comment by jonquag** [comment]
> Do you know why so many files were modified in this PR?

does this have to do with this branch being originally the demo branch?

**Comment by jessdesr** [comment]
> > Do you know why so many files were modified in this PR?
> 
> does this have to do with this branch being originally the demo branch?

Yeah, it would.  You'll most likely want to rebase off of dev.  This a bit more of an experienced git practice, so I'll outline it for you:

1) Checkout dev and pull to get latest changes
2) Checkout this branch
3) Command: `git rebase dev`
4) If there are any conflicts while you're rebasing, just message me and we can work through it together

File: client/src/components/Forum/PostCard.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    postDateInfo: {
+        paddingTop: 10,
+    },
+}));
+
+const PostCard = props => {
+    const classes = useStyles();
+
+    return (
+        <Card className={classes.root}>
```
Comment (timegambit) [inline client/src/components/Forum/PostCard.js]: I think this component needs a little more room for long text. In the image below the "Qualification" on the Google analytics post is right at the border. Maybe just adding a paddingRight would work. 

Screen Shot 2021-01-28 at 9 10 46 AM

Also I was wondering how you feel about making this a Paper component with an elevation. I think it looks more like the mockup. And I think there's a way to customize the shadow of the paper to give it that light blueish look. What is looks like using Paper with elevation=5:

Screen Shot 2021-01-28 at 9 13 17 AM

File: client/src/components/Forum/ForumContent.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                    <Typography variant="h1" style={{ paddingLeft: '2em' }}>
+                        Forum
+                    </Typography>
+                </Grid>
+
+                <Grid item container sm={3}>
+                    <Button
+                        className={classes.button}
+                        onClick={addPost}
+                        variant="text"
```
Comment (timegambit) [inline client/src/components/Forum/ForumContent.js]: I think variant="text" by default, so you don't need to include this line on buttons.

Comment (jonquag) [reply client/src/components/Forum/ForumContent.js]: > I think variant="text" by default, so you don't need to include this line on buttons.

variant="text"
color="primary"

Are setting the button to the override in theme.js
Not sure if leaving it out would render that override button, but I can take a look

Comment (timegambit) [reply client/src/components/Forum/ForumContent.js]: I tested it out and it looked like it did!

File: client/src/components/Forum/PostCard.js (hunk 1 above)
Comment (jonquag) [reply client/src/components/Forum/PostCard.js]: Good spot on the right padding. I agree with you. I also thought the text size in the sketch file of 22 might be a bit large.  

The Paper idea is a great idea with the elevation. I'll look into this today.

**Comment by jonquag** [comment]
forum post cards with paper

Here's an updated look at the forum PostCard.js with paper and the paddingRight added. Great spot,  @timegambit

**Comment by timegambit** [comment]
> forum post cards with paper
> 
> Here's an updated look at the forum PostCard.js with paper and the paddingRight added. Great spot, @timegambit

Looks great Jon!

**Comment by jessdesr** [comment]
I think I might set the width of the posts to be proportionate to the size of the container, with a larger maxWidth set and maybe a degree of padding?  For example, check the padding in the mockup (even though the mockups are a pretty weird aspect ratio) and see if you can set width to 100% and padding to match the mockup.  

You can test how it would show up on larger screens by using the magnification function in chrome.   We don't want it to get ridiculously long though so set a maxWidth with your best judgement

**Review by jessdesr — APPROVED** [review]
Looks good, nice work on the buttons

**Review by jonquag — COMMENTED** [review]
> I think I might set the width of the posts to be proportionate to the size of the container, with a larger maxWidth set and maybe a degree of padding? For example, check the padding in the mockup (even though the mockups are a pretty weird aspect ratio) and see if you can set width to 100% and padding to match the mockup.
> 
> You can test how it would show up on larger screens by using the magnification function in chrome. We don't want it to get ridiculously long though so set a maxWidth with your best judgement

I can look into this.

**Review by ethanrosenberg — APPROVED** [review]
Great work Jon! The paper addition looks really good

File: client/src/components/Forum/PostCard.js (hunk 1 above)
Comment (amha19) [reply client/src/components/Forum/PostCard.js]: The branch I am checking has the right padding and the cards look great. 
If you try using 'rem' for font sizes let me know if you are getting any errors.

**Pull request merged**